addon/-private/environment/index.js

```diff
@@ -31,7 +31,9 @@ export default class Environment extends EmberObject.extend(Evented) {
       object = this;
     }
 
-    const resolveFieldMeta = this.__resolveFieldMeta__;
+    const resolveFieldMeta = typeof this.__resolveFieldMeta__ === 'function' ?
+      this.__resolveFieldMeta__ :
+      () => {};
```

Comment:
could this go in the `constructor` method as the default value for the `metaForField` param?

Reply:
@deverstalmage nice, i like that.  fixed in https://github.com/salsify/ember-exclaim/pull/13/commits/055779cd6e425f23e0569d458e99b187da2fd80b